### PR TITLE
Change calling command to dynamic command

### LINKED ISSUE #17: Make launch command dynamic
In the current, when `/hey` is pushed, production app, dev-1, dev-2 and dev-3 respond.

before
```bash
/hey

kimetaro says "おーきに”
kimetaro-dev1 says "おーきに”
kimetaro-dev2 says "おーきに”
kimetaro-dev3 says "おーきに”
```

after
```bash
/hey
kimetaro says "おーきに”

/hey-dev1
kimetaro-dev1 says "おーきに”
```

### PR DESCRIPTION
**Pull request by kamontia**
The code for resolving this issue is already implemented.

Unfortunately, some implementation is running on Heroku or somewhere.
So the all app in old process returns something even if `/hey-dev1` is pushed.

What do you think about prefixing instead of suffixing?
suffix: `/hey-dev1`
prefix: `/dev1-hey`

Close #17 
Ref: #7

### CODE REVIEW

File: kimetaro.py [hunk 1]
```diff
@@ -7,6 +7,9 @@
 import discord
 import pysnooper
 
+# Value initialization
+CALLCOMMAND_SUFFIX = ''
```
Comment (chaspy) [inline kimetaro.py]: The name `CALLCOMMAND_SUFFIX` is not good.
(Two words are combined)

How about `COMMAND_SUFFIX` or `MESSAGE_SUFFIX` ?
As indicated by the `message.content.startswith` method, in discord.py `CALLCOMMAND` stands for `message`.

Comment (kamontia) [reply kimetaro.py]: sed -i '' s/CALLCOMMAND_SUFFIX/COMMAND_SUFFIX/g kimetaro.py

**Review by kamontia — COMMENTED** [review]
Please check and merge

**Review by chaspy — APPROVED** [review]
LGTM 👍

**Pull request merged**